main.py: Don't continue upload looping if the login session has expired.

TerraTech opened the pull request:
This fixes a problem I had when I was uploading 50 1GB files, and on the 8th one the session had expired.  I didn't see this condition _(distracted)_ until it was on the 35th file.  During that time, the loop had racked up a lot of time from performing the _(unused)_ MD5 calculations.

In summation, it should stop the loop immediately when there is a "need verify" condition.

TerraTech committed:
main.py: Don't continue upload looping if the login session has expired.

dnigamer commented:
Well I haven't got around to test the split upload functionality to that extent mainly because i can upload files quick enough while the session is still valid 😅
Additionally, I probably wouldn't use the `sys.exit()` just like that because I want things to be a little more verbose and not have the program to "crash" on a session expired exception.

dnigamer committed:
main.py: removed sys.exit() and replaced it with friendly exit

By setting the errors to True and breaking the for loop that does the uploading procedure, it makes the program exit gracefully and verbosely.

dnigamer committed:
main.py: Removed unused import and renamed returned value.
